src/conversations/Conversations.ts

```diff
@@ -108,28 +112,35 @@ export default class Conversations {
   }
 
   private async listV2Conversations(): Promise<Conversation[]> {
-    return this.v2Cache.load(async ({ latestSeen }) => {
-      const newConvos: Conversation[] = []
-      const invites = await this.client.listInvitations({
-        startTime: latestSeen
-          ? new Date(+latestSeen - CLOCK_SKEW_OFFSET_MS)
-          : undefined,
-        direction: SortDirection.SORT_DIRECTION_ASCENDING,
-      })
+    return this.v2Cache.load(async ({ latestSeen }) =>
+      this.v2ConversationLoader(latestSeen)
+    )
+  }
 
-      for (const sealed of invites) {
-        try {
-          const unsealed = await sealed.v1.getInvitation(this.client.keys)
-          newConvos.push(
-            await ConversationV2.create(this.client, unsealed, sealed.v1.header)
-          )
-        } catch (e) {
-          console.warn('Error decrypting invitation', e)
-        }
+  // Callback called in listV2Conversations and in newConversation
+  private async v2ConversationLoader(
```

Comment:
I needed to break this into its own function to avoid a deadlock that would be caused by calling `listV2Conversations` from inside a conversation loader.